src/i18n.ts

```diff
@@ -1047,6 +1059,12 @@ const en: Record<I18N, string> = {
   [I18N.settingsSecurity]: 'Security',
   [I18N.sent]: 'Sent',
   [I18N.transactionSendTitle]: 'Sent',
+  [I18N.transactionReceiveTitle]: 'Received',
+  [I18N.transactionAmountText]: `{{prefix}} {{value}} ${CURRENCY_NAME}`,
```

Comment:
can be divided into I18N.transactionPositiveAmountText and I18N.transactionNegativeAmountText